test/src/components/campaign/CampaignInfoTest.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { fetch } from 'whatwg-fetch';
+import { shallow } from 'enzyme';
+import { Col } from 'react-bootstrap';
+import CampaignInfo from '../../../../src/components/campaign/campaignInfo';
+
+const mockInfo = {
```

Comment:
nice, this approach could work for client calls; supply them with a request body in a shape that's know so the expect could be tested.